01-node-tutorial/answers/prompter.js

```diff
@@ -21,13 +21,13 @@ const getBody = (req, callback) => {
 };
 
 // here, you could declare one or more variables to store what comes back from the form.
-let item = "Enter something below.";
+let item = "Welcome to Stephanie's page";
 
 // here, you can change the form below to modify the input fields and what is displayed.
 // This is just ordinary html with string interpolation.
 const form = () => {
   return `
-  <body>
+  <body style="background-color:red;">
   <p>${item}</p>
   <form method="POST">
   <input name="item"></input>
```

Comment:
Can just add a line(before line 33)<p>Enter a number</p>, for user to understand what input is it expecting